numpy/core/einsumfunc.py

```diff
@@ -763,14 +764,15 @@ def einsum_path(*operands, **kwargs):
     of the contraction and the remaining contraction ``(0, 1)`` is then
     completed.
 
+    >>> np.random.seed(123)
     >>> a = np.random.rand(2, 2)
     >>> b = np.random.rand(2, 5)
     >>> c = np.random.rand(5, 2)
     >>> path_info = np.einsum_path('ij,jk,kl->il', a, b, c, optimize='greedy')
     >>> print(path_info[0])
     ['einsum_path', (1, 2), (0, 1)]
     >>> print(path_info[1])
-      Complete contraction:  ij,jk,kl->il
```

Comment:
This seems suspiscious to me

Reply:
The contraction is indeed constant in this case, but the FLOP counts don't seem to be fully reproducible or at least fully portable, which is maybe not surprising? The `# may vary` designation may seem to instill a lack of confidence in the contraction, but it was probably intended for the FLOP count variability (the entire block is labelled as variable as a result).